etna/models/nn/rnn.py

```diff
@@ -244,6 +298,8 @@ def __init__(
             number of layers
         hidden_size:
             size of the hidden state
+        embedding_sizes:
```

Comment:
I don't see any comments about handling of embeddings:
- Working with label-encoded values
- Handling unknown values

Reply:
Added